example/main.tf

```diff
@@ -19,26 +19,27 @@ terraform {
   required_version = "~> 0.11"
 
   # Use a GCS Bucket as a backend
-  backend "gcs" {}
+  backend "gcs" {
+  }
```

Comment:
While I prefer this multiline syntax, the `verify-terraform.sh` script currently uses `sed` to comment it out so we don't have to use a GCS bucket for the automated tests. Doing multiline stuff with `sed` looks a lot more complicated, and I'm not aware of any other good ways to comment this out in a script. So for now I think we should leave it on one line.

I would in the long run like to actually use a GCS bucket for the automated testing, as part of #34.